src/Altinn.App.Core/Internal/Process/ProcessNavigator.cs

```diff
@@ -37,7 +37,11 @@ ILogger<ProcessNavigator> logger
     public async Task<ProcessElement?> GetNextTask(Instance instance, string currentElement, string? action)
     {
         List<ProcessElement> directFlowTargets = _processReader.GetNextElements(currentElement);
-        List<ProcessElement> filteredNext = await NextFollowAndFilterGateways(instance, directFlowTargets, action);
+        List<ProcessElement> filteredNext = await NextFollowAndFilterGateways(
+            instance,
+            directFlowTargets.Cast<ProcessElement?>().ToList(),
```

Comment:
```C#
List<ProcessElement?> directFlowTargets = _processReader.GetNextElements(currentElement)!;
List<ProcessElement> filteredNext = await NextFollowAndFilterGateways(instance, directFlowTargets, action);
```
No reason to make a copy

Reply:
You mentioned if a direct cast would be better/clearer, to which I agree.. But surprisingly that does produce a warning 
![image](https://github.com/Altinn/app-lib-dotnet/assets/5425986/5b11c488-7594-41f2-a6ea-e2e682937d27)